README.md

```diff
@@ -283,6 +209,4 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/212623559-9ab438ee-c687-45c0-a22d-a6c962757de6.png)

- [ ] Your license file is looking like a template. Kindly add your **name**, **current year** and **app-type** in the license file to make your project licensed :+1: